<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>subWindow</class>
 <widget class="QMainWindow" name="subWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>365</width>
    <height>169</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnSubClose">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>40</y>
      <width>181</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Close</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
